.github/workflows/tests.yml

```diff
@@ -0,0 +1,53 @@
+name: Tests
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+
+  build_cluster:
+    name: Redis EE (${{ matrix.type }})
+    runs-on: ubuntu-latest
+
+    strategy:
+      fail-fast: false
+      matrix:
+        type:
+          - dmc
+          - oss
+
+    steps:
+
+      - name: Install Docker-compose
+        run: |
+          DOCKER_CONFIG=${DOCKER_CONFIG:-$HOME/.docker}
```

Comment:
docker-compose should just exist in github actions automatically, and since we're not version tied by this, maybe it's unncessary?